### DISCUSSION TITLE
Help to install Pybricks

### DISCUSSION THREAD

**Discussion by Darksail9** [pybricks/support#400] [Q&A]
I cannot connect the technician hub, after clicking the bluetooth icon, an endless search, the connect button is gray.  what can be done so that the hub is detected by the program?  device-technician hub and smartphone.

**Comment by dlech** [comment]
Did you install the Pybricks firmware first? https://pybricks.com/install/technic-boost-city/

**Reply by Darksail9** [reply]
Нет,хаб нужно подключить,у меня не получается. Объясните пожалуйста как можно подключить? Заранее премного благодарен!

> No, the hub needs to be connected, I can't. Please explain how you can connect? Thanks a lot in advance!

**Reply by dlech** [reply]
The link explains everything. It even has videos to show you what to do.

**Reply by Darksail9** [reply]
Проделал все с абсолютной точностью-выскакивает ошибка что хаб долго не отвечает. На форуме нашел топик с данной проблемой,но там указаны виндовс и мак ос,я же использую андроид. Что ещё сделать чтобы установить пайбрикс и возможность установки прошивок?

> I did everything with absolute precision - an error pops up that the hub does not respond for a long time. I found a topic on the forum with this problem, but there are Windows and Mac OSs, but I use android. What else can I do to install the paibrix and the ability to install firmware?

**Comment by dlech** [comment]
Which hub are you using? Be sure to unplug everything from the hub first and make sure the batteries are fresh.

**Reply by laurensvalk** [reply]
Related: https://github.com/pybricks/support/issues/329#issuecomment-857617644

That's for city hub with Android, though I think OP has Technic Hub. I can try to confirm early next week.

**Reply by Darksail9** [reply]
Специально сегодня купил новые батареи энерджайзер для техник хаба и Сити хаба, откатил оба хаба на стоковые прошивки и провел все манипуляции с начала- зажал кнопку не отпуская, выбрал Лего бутлоадер,дождался когда индикатор замигает розовым,потом разными цветами, потом потухнет,снова замигает разными цветами отпустил кнопку, дождался конца загрузки-итог- уведомление что хаб задержался с ответом. У меня андроид устройства может быть проблема в этом?

**Reply by Darksail9** [reply]
Especially today I bought new energizer batteries for the hub and City hub technicians, rolled back both hubs to the stock firmware and carried out all the manipulations from the beginning - held down the button without releasing it, chose the Lego bootloader, waited until the indicator blinks pink, then in different colors, then goes out, blinks again  released the button in different colors, waited for the end of the download, the result, a notification that the hub was delayed with a response.  I have an android device may have a problem with this?
